vipps-recurring-api.md

```diff
@@ -752,83 +775,58 @@ to set up a new agreement.
 
 ### Charge states
 
-![Recurring charge states](images/Recurring-chargestates-new.svg)
-
-The illustration above is a simplification. This table has all the details for
-the charge states returned by
+This table has all the details for the charge states returned by
 [`GET:/recurring/v2/agreements/{agreementId}/charges/{chargeId}`](https://vippsas.github.io/vipps-recurring-api/#/Charge%20Endpoints/getCharge):
 
-| # | State      | Description                                                                          |
-|:--|:-----------|:-------------------------------------------------------------------------------------|
-| 1 | `PENDING`  | The charge has been created, but is not yet visible to the user in Vipps. |
-| 2 | `DUE`      | The charge is now visible in Vipps and will be processed on the due date. |
-| 3 | `CHARGED`  | The charge has been completed. |
-| 4 | `FAILED`   | The charge has failed for some reason, i.e. expired card, insufficient funds, etc. Read the [Charge failure reasons](https://github.com/vippsas/vipps-recurring-api/blob/master/vipps-recurring-api.md#charge-failure-reasons) section for more details. |
-| 5 | `REFUNDED` | The charge has been refunded. The timeframe for issuing a refund is 365 days from the date of capture. |
-| 6 | `PARTIALLY_REFUNDED`| A part of the captured amount has been refunded. |
-| 7 | `RESERVED` | An initial charge with `transactionType` set to `RESERVE_CAPTURE` changes state to `CHARGED` when captured successfully. |
-| 8 | `CANCELLED` | The charge has been cancelled. |
-| 9 | `PROCESSING` | The charge is currently being processed by Vipps. Normal processing takes less than 1 second, but in some cases they can stay in this status for several minutes |
+| State      | Description                                                                          
+|:-----------|:-------------------------------------------------------------------------------------
+| `PENDING`  | The charge has been created, but _may_ not yet be visible in Vipps. **Please note:** All charges due in 30 days or less are visible in Vipps.|
+| `DUE`      | The charge is visible in Vipps and will be processed on the due date. This status is just a quick transition from `PENDING` to `PROCESSING`. |
```

Comment:
THe charge is DUE for up to 30 days before the due-date, so this is NOT a quick transition

Reply:
Fixed, thanks